src/utils/test-utils/testConstants.js

```diff
@@ -0,0 +1,8 @@
+export const testInputNames = Object.freeze({
```

Comment:
It seems like this object holds constants specifically for checkbox related input names, the variable name should reflect that. 
Or if it's for **any** input element names then i'd imagine we'd want the current properties within it to mention 'checkbox' otherwise it will be hard to distinguish when adding other unrelated ones.

Reply:
These are all test ID constants used for the date boxes at the moment, put here rather than mixing them up with the UI constants.